src/bsl/features/signatureHelpProvider.ts

```
@@ -1,58 +1,132 @@
+function currentSymbol(model: editor.ITextModel, position: number): Constructor | MethodCall | undefined {
+    const scope = EditorScope.getScope(model)
+    const node = scope.getAst().getCurrentNode(position)
+    if (node) {
+        return resolveMethodSymbol(node)
+    }
+}
+
+async function createConstructorSignatures(symbol: Constructor): Promise<languages.SignatureHelp | undefined> {
+    const typeId = await symbol.getResultTypeId()
+    if (typeId) {
+        const constructor = GlobalScope.getConstructor(typeId)
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)